README.md

```diff
@@ -437,6 +437,9 @@ Run `make test` to invoke the test suite, found in the test-suite subdirectory.
 - Iulia Tamas
 - Andrew Twyman
 
+## Contributors
```

Comment:
Hmm, yeah, maybe we should think about having an official list.  This is the first time someone's raised the issue of a policy like this.  Is the policy "you must add to the list" or can it be interpreted as "add to the list if it exists"?  It seems like in our current state our "contributors list" is `git log`, and I don't know if there are global norms on GitHub about doing otherwise.